README.md

```diff
@@ -71,19 +71,39 @@ Dialogue:
           - Bot: <WORD>
 ```
 
+Handling multiple attachments
+
+```YAML
+Title: Conversation with rich messages
+Dialogue:
+  - Bot: Hi
+  - Human: Hello
+  - Human: How are we doing this month?
+  - Bot: "Month-to-date: <IMAGE> <CARDS>"
+  - Human: Show me ad spend compared to last month
+  - Bot: <IMAGE> <IMAGE>
+```
+
 ## Special Tags
 
-You can use the following tags in your test dialogue files:
+You can use the following tags anywhere in the bot’s response:
 
 Tag | Meaning
 --- | ---
 `<*>` | Matches anything, including whitespaces
 `<WORD>` | A single word without whitespaces
-`<IMAGE>` | An image attachment
-`<CARDS>` | A card attachment
 `<(REGEX)>` | Any regex expression, i.e. `<([0-9]{2})>`
 `<$VARNAME>` | An expression from the locale/translation file
 
+You can match one or more attachments **on their own, or at the end** of the
+response:
+
+Tag | Meaning
+--- | ---
+`<CARDS>` | Any card attachment
```

Comment:
Thanks for the docs here. I would just add a comment here that <CARDS> includes buttons, as this might not be obvious to them.